spec/models/user_spec.rb

```diff
@@ -297,4 +297,25 @@
       expect { do_purge }.to change(User, :count).by(0)
     end
   end
+
+  #sam Unit test
+  describe User do
+    let(:emails) {  ['a@a.com','b@b.com'] }
+    let(:users) { double('Users') }
+        it '.superadmin_emails' do 
+          expect(users). to receive(:pluck).with(:email).and_return(emails)
+          expect(User).to receive(:superadmins).and_return(users)
+          expect(User.superadmin_emails).to eq emails
+        end
+  end
+
+  #integration test 
```

Comment:
@tansaku. Yes, I would like to keep the placeholder in place. Do you want me to remove the Unit test? Trying to understand the login behind superadmin account. So I created a phony org and tried to see if the email was generated to the superadmin. It did. agreed on that?